doc/OneStepSetup.md

```diff
@@ -83,6 +83,7 @@ Given that the Pi contains sensitive information like your home wifi password an
 
 ### Troubleshooting
 
+* If everything seems to be working, but you still don't see the USB drive(s) either on your local machine, or in the car, check that you are indeed using a USB data cable, and not a charge-only cable. Also ensure you are plugged into the USB port on the Raspberry PI, and not the power port.
```

Comment:
It seems a bit odd to start the "troubleshooting" section with "if everything seems to be working ...".

It might be better to fold (most of) this section into the troubleshooting page on the wiki, and just link there (the wiki page already mentions cable issues).